### PR TITLE
AP-5427 Add aux custom element

### PR DESCRIPTION
**Pull request by dreamind**

### CODE REVIEW

File: Apromore-Core-Components/Apromore-BPMN-Editor/Apromore-BPMN-Modeler-JS/app/provider/entries/aux/AuxEntries.js [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+      { values: [] }, element, bpmnFactory);
+    element.extensionElements = extensionElements;
+  }
+
+  return extensionElements;
+}
+
+function getImg(element, bpmnFactory) {
+  var extensionElements = getExtensionElements(element, bpmnFactory);
+//  var img = extensionElements.values.filter(function(e) {
```
Comment (Janite) [inline Apromore-Core-Components/Apromore-BPMN-Editor/Apromore-BPMN-Modeler-JS/app/provider/entries/aux/AuxEntries.js]: Can we remove this comment from the code?

**Review by Janite — CHANGES_REQUESTED** [review]
Images can be added and resized as expected. Attachment tab looks good. Just have a few comments/questions:

1. Is there a way to remove icons and uploaded images once they have been added?
2. A link added to an element redirects to a page in apromore portal instead of the direct link text added. Wanted to check if this is intended.
3. The white background of the x's used to clear the link text and url overlap the text input border.

File: Apromore-Core-Components/Apromore-BPMN-Editor/Apromore-BPMN-Modeler-JS/app/provider/entries/aux/AuxEntries.js (hunk 1 above)
Comment (dreamind) [reply Apromore-Core-Components/Apromore-BPMN-Editor/Apromore-BPMN-Modeler-JS/app/provider/entries/aux/AuxEntries.js]: Feedback have been implemented

**Review by Janite — APPROVED** [review]
Looks good and works as expected.

**Pull request merged**